examples/tests/igraph_closeness.c

```diff
@@ -0,0 +1,287 @@
+#include <igraph.h>
+#include <stdio.h>
+#include <math.h>
+
+/* helper function, useful for viewing output */
+void igraph_vector_t_print(igraph_vector_t *exp, igraph_vector_t *actual, int size) {
+    for (int i = 0; i < size; i++) {
+        double res = VECTOR(*exp)[i];
+        double res_2 = VECTOR(*exp)[i];
+        printf("%lf %lf\n", res, res_2);
+    }
+    printf("\n");
+}
+
+int igraph_check_vectors_equal(igraph_vector_t *v, igraph_vector_t *w, int size) {
+    for (int i = 0; i < size; i++) {
+        int v1 = 1000000 * VECTOR(*v)[i];
+        int v2 = 1000000 * VECTOR(*w)[i];
+
+        if (v1 != v2) {
+            /* taking into account rounding up/down issues
+            e.g. for intermediate case no weights
+            100000 == 99999 */
+            if (v2+1 != v1) {
+                return 0;
+            }
+        }
+    }
+    return 1;
+}
+
+int simple_test_case_no_weights_undirected() {
```

Comment:
Since the test functions don't actually return anything (the return value, which is always `0`, is never used), make them `void`.